### PR TITLE
Workshop

### PR DESCRIPTION
**Pull request by przmv**
This PR adds the actual workshop materials

TODO:

- [ ] Use `main` branch in **Open in Cloud Shell** button in `README.md`

### CODE REVIEW

**Review by SamDesmondKing — COMMENTED** [review]
Looks awesome Petr. One small suggestion, maybe you could add the command to add the machine stats script location to the PATH? 

image

**Review by jmariomejiap — APPROVED** [review]
It looks good to me @przmv. It is nice to get this workshop content ready for future uses 👍

**Comment by przmv** [comment]
On Wed, 2021-08-11 at 07:25 -0700, Sam King wrote:
> @SamDesmondKing commented on this pull request.
> Looks awesome Petr. One small suggestion, maybe you could add the
> command to add the machine stats script location to the PATH?

Thanks, Sam!

Actually, it should be added. I'll check.

File: machine-stats.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+Open your **Tidal Migrations Platform** and search for one of the hosts on the
+**Assess/Servers** page.
+
+## Clean up
+
+To avoid incurring charges to your Google Cloud account for the resources used
+in this tutorial, follow this steps.
+
+```bash
+gcloud deployment-manager deployments delete instance-pool
```
Comment (jmariomejiap) [inline machine-stats.md:172]: Nice, 👍

File: machine-stats.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+
+Do you want to continue (y/N)?
+```
+
+The deployment and the resources you created are permanently deleted.
+
+Alternatively, you can delete your Cloud project to stop billing for all the
+resources used within that project.
+
+**Caution**: Deleting a project has the following effects:
```
Comment (jmariomejiap) [inline machine-stats.md:189]: good idea to add this extra section 👍

**Review by jmariomejiap — APPROVED** [review]
now, It's even better. It looks ready to be released @przmv 🚀

**Comment by surmistry** [comment]
Just ran through it again 👍🏽 
Great job making a good flow into a seamless, clean experience

**Pull request merged**